stdlib/3/sys.pyi

```diff
@@ -5,120 +5,132 @@
 
 from typing import (
     List, NoReturn, Sequence, Any, Dict, Tuple, TextIO, overload, Optional,
-    Union, TypeVar, Callable, Type,
+    Union, TypeVar, Callable, Type
 )
 import sys
-from types import FrameType, TracebackType
+from types import FrameType, ModuleType, TracebackType
+
+if sys.version_info >= (3, 3):
+    from importlib.abc import MetaPathFinder
+else:
+    MetaPathFinder = Any
```

Comment:
This seems technically wrong (it will expose `sys.MetaPathFinder` on 3.2), but we don't support 3.2 anyway.

Reply:
Good, I've dropped conditional import.
BTW what is minimal supported Python version?

Reply:
Oops, PR was merged. Next time than.

Reply:
It's 2.7 and 3.3+: see https://github.com/python/typeshed#about.